doc/nrf/templates/application_README.rst

```diff
@@ -0,0 +1,263 @@
+   * You need not list all possible configuration options, but only the ones that are relevant.
+
+Check and configure the following library options that are used by the application:
+
+* :kconfig:option:`CONFIG_PDN_DEFAULT_APN` - Used for manual configuration of the Access Point Name (APN).
+* :kconfig:option:`CONFIG_MODEM_ANTENNA_GNSS_EXTERNAL` - Selects an external GNSS antenna.
+
+Configuration files*
+====================
+
+.. note::
+   Add this section if the application provides predefined configuration files.
+
+The application provides predefined configuration files for typical use cases.
+You can find the configuration files in the :file:`XXX` directory.
+
+The following files are available:
+
+* :file:`filename.conf` - Specific scenario.
+  This configuration file ...
+
+Building and running
+********************
+
+.. note::
+   * Include the standard text for building - either ``.. include:: /includes/application_build_and_run.txt`` or ``.. include:: /includes/application_build_and_run_ns.txt`` for the build targets that use :ref:`Cortex-M Security Extensions <app_boards_spe_nspe>`.
+   * The main supported IDE for |NCS| is |VSC|, with the |nRFVSC| installed.
+     Therefore, build instructions for the |nRFVSC| are required.
+     Build instructions for the command line are optional.
+   * See the link to the `nRF Connect for Visual Studio Code`_ documentation site for basic instructions on building with the extension.
+   * If the application uses a non-standard setup, point it out and link to more information, if possible.
+
+.. |application path| replace:: :file:`applications/XXX`
+
+.. include:: /includes/application_build_and_run.txt
+
+Some title*
+===========
+
+.. note::
+   If required, add subsections for additional build instructions.
+   Use these subsections sparingly and only if the content does not fit into other sections (mainly `Configuration*`_).
+   If the additional build instructions are valid for other applications as well, consider adding them to the :ref:`getting_started` section instead and linking to them.
+
+Testing
+=======
+
+|test_application|
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Reset the development kit.
+#. And so on ...
+
+.. note::
+   * Use the shortcuts provided in :file:`doc/nrf/shortcuts.txt` to keep the wording consistent.
+   * If there are different ways of testing, introduce them in this section (for example, "After programming the application to your development kit, you can test it either by ...") and add subsections for the different scenarios.
+
+Application output*
+===================
+
+.. note::
+   Add the full output of the application in this section or include parts of the output in the testing steps in the previous section.
+
+The application shows the following output:
+
+.. code-block:: console
+
+   [00:00:02.029,174] <inf> zigbee_app_utils: Zigbee stack initialized
+
+Applications and samples*
+*************************
```

Comment:
Should we modify this heading as 'Implementation in other applications and samples'